src/AddTodoForm.js

```diff
@@ -15,9 +17,14 @@ function AddTodoForm({onAddTodo}) {
       
     };
     return (
-        <form onSubmit={handleAddTodo}>        <label htmlFor="todoTitle">Title</label>
-            <input type="text" id="todoTitle" name="title" value={todoTitle} onChange={handleTitleChange}></input>
-
+        <form onSubmit={handleAddTodo}>  
+        <InputWithLabel
+        todoTitle={todoTitle} 
+        handleTitleChange={handleTitleChange}
+        >
+            Title
+        </InputWithLabel>
+              
             <button type="submit">Add</button>
```

Comment:
There are a couple of minor indent issues here, for example the properties inside the `<InputWithLabel>` tag should all be indented further, and the `<button>` doesn't need to be indented that far.